src/routes/api/getAppData/+server.ts

```diff
@@ -1,24 +1,23 @@
-import { json } from '@sveltejs/kit';
-import { getUserActiveNotifications } from '$lib/server/server.js';
+import { json, type RequestEvent } from '@sveltejs/kit';
 import { getReservationsSince } from '$lib/server/reservation';
 import { getAllUsers } from '../../../lib/server/user';
 
-export async function POST({ request }) {
+// TODO: break this apart into separate functions
```

Comment:
doesn't it slow things down even further to break these into multiple POST calls?

Reply:
by grouping them together, the endpoints average speed is equivalent to the worst performing function.

but if you separate them and do something asynchronously per single purpose function, it finishes the specific function the data serves immediately.

Reply:
ahh ok that makes sense